### DISCUSSION TITLE
Best way to put GRBL into +ve coordinate space.

### DISCUSSION THREAD

**Discussion by HattonLe** [fra589/grbl-Mega-5X#218] [Q&A] [answered]
What is the best way to set the CNC machine into positive coordinate space. I would like to do this because it would a) match up with my milling machine and b) match up with the Dxf2Gcode application (which seems to be only +ve coordinate space), which I then feed into the CAMotics simulator (not sure if CAMotics handles -ve coordinate space).
(I'm currently using HOMING_FORCE_SET_ORIGIN so my machine has (0,0) as its homed work origin off the limit switches).

**Comment by langwadt** [comment]
use work coordinates (G54 to G59) instead of machine coordinates

**Comment by fra589** [comment, marked as answer]
Hi @casiobearing,  
  
You should read the theory about coordinates systems on CNC milling machine:  
http://linuxcnc.org/docs/2.8/html/gcode/coordinates.html

@++;
Gauthier.
